types/durableClient.d.ts

```diff
@@ -350,6 +350,16 @@ export interface StartNewOptions {
      * JSON-serializable input value for the orchestrator function.
      */
     input?: unknown;
+
+    /**
+     * The traceparent header value to use for the new orchestration instance.
+     */
+    traceParent?: string;
+
+    /**
+     * The tracestate header value to use for the new orchestration instance.
+     */
+    traceState?: string;
```

Comment:
Had an offline discussion with Chris and I'm looking to see if we can remove these properties by setting the active context to the new span in the client function and reading that in `startNew()`.